### ISSUE TITLE
[release/2.11][wsl] timm_models_accuracy_training_bfloat16 convnextv2_nano.fcmae_ft_in22k_in1k fail_accuracy

### ISSUE THREAD

**Issue by bjarzemb** [intel/torch-xpu-ops#2952]
### 🐛 Describe the bug

PyTorch 2.10:
```
loading model: 0it [00:00, ?it/s]
loading model: 0it [00:04, ?it/s]
xpu  train convnextv2_nano.fcmae_ft_in22k_in1k 
pass
```

PyTorch 2.11:
```
loading model: 0it [00:00, ?it/s]
loading model: 0it [00:04, ?it/s]
xpu  train convnextv2_nano.fcmae_ft_in22k_in1k 
E0225 06:11:14.471000 62810 site-packages/torch/_dynamo/utils.py:3393] RMSE (res-fp64): 0.00228, (ref-fp64): 0.00025 and shape=torch.Size([160]). res.dtype: torch.bfloat16, multiplier: 3.000000, tol: 0.010000, use_larger_multiplier_for_smaller_tensor: 0
E0225 06:11:14.471000 62810 site-packages/torch/_dynamo/utils.py:3231] Accuracy failed for key name stages.2.downsample.0.weight.grad
fail_accuracy
```

### Versions

PyTorch version: 2.11.0+xpu
Is debug build: False
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: Ubuntu 24.04.1 LTS (x86_64)
GCC version: (Ubuntu 13.3.0-6ubuntu2~24.04) 13.3.0
Clang version: Could not collect
CMake version: version 4.2.1
Libc version: glibc-2.39

Python version: 3.12.12 | packaged by conda-forge | (main, Jan 26 2026, 23:51:32) [GCC 14.3.0] (64-bit runtime)
Python platform: Linux-6.6.87.2-microsoft-standard-WSL2-x86_64-with-glibc2.39
Is CUDA available: False
CUDA runtime version: No CUDA
CUDA_MODULE_LOADING set to: N/A
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
Is XPU available: True
XPU used to build PyTorch: 20250302
Intel GPU driver version:
* libze1:       1.26.2-1~24.04~ppa1
* intel-opencl-icd:     26.01.36711.4-1~24.04~ppa1
Intel GPU models onboard:
N/A
Intel GPU models detected:
* [0] _XpuDeviceProperties(name='Intel(R) Graphics [0xe20b]', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero V2', type='gpu', device_id=0xE20B, uuid=86800be2-0000-0000-0400-000000000000, driver_version='1.14.36711+4', total_memory=11869MB, local_mem_size=128KB, max_compute_units=160, gpu_eu_count=160, gpu_subslice_count=20, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True
Caching allocator config: N/A

CPU:
Architecture:                         x86_64
CPU op-mode(s):                       32-bit, 64-bit
Address sizes:                        46 bits physical, 48 bits virtual
Byte Order:                           Little Endian
CPU(s):                               20
On-line CPU(s) list:                  0-19
Vendor ID:                            GenuineIntel
Model name:                           Intel(R) Core(TM) Ultra 7 265K
CPU family:                           6
Model:                                198
Thread(s) per core:                   1
Core(s) per socket:                   20
Socket(s):                            1
Stepping:                             2
BogoMIPS:                             7756.81
Flags:                                fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss ht syscall nx pdpe1gb rdtscp lm constant_tsc rep_good nopl xtopology tsc_reliable nonstop_tsc cpuid tsc_known_freq pni pclmulqdq vmx ssse3 fma cx16 pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch ssbd ibrs ibpb stibp ibrs_enhanced tpr_shadow ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid rdseed adx smap clflushopt clwb sha_ni xsaveopt xsavec xgetbv1 xsaves avx_vnni vnmi umip waitpkg gfni vaes vpclmulqdq rdpid movdiri movdir64b fsrm md_clear serialize flush_l1d arch_capabilities
Virtualization:                       VT-x
Hypervisor vendor:                    Microsoft
Virtualization type:                  full
L1d cache:                            960 KiB (20 instances)
L1i cache:                            1.3 MiB (20 instances)
L2 cache:                             60 MiB (20 instances)
L3 cache:                             30 MiB (1 instance)
NUMA node(s):                         1
NUMA node0 CPU(s):                    0-19
Vulnerability Gather data sampling:   Not affected
Vulnerability Itlb multihit:          Not affected
Vulnerability L1tf:                   Not affected
Vulnerability Mds:                    Not affected
Vulnerability Meltdown:               Not affected
Vulnerability Mmio stale data:        Not affected
Vulnerability Reg file data sampling: Not affected
Vulnerability Retbleed:               Mitigation; Enhanced IBRS
Vulnerability Spec rstack overflow:   Not affected
Vulnerability Spec store bypass:      Mitigation; Speculative Store Bypass disabled via prctl
Vulnerability Spectre v1:             Mitigation; usercopy/swapgs barriers and __user pointer sanitization
Vulnerability Spectre v2:             Mitigation; Enhanced / Automatic IBRS; IBPB conditional; RSB filling; PBRSB-eIBRS SW sequence; BHI Not affected
Vulnerability Srbds:                  Not affected
Vulnerability Tsx async abort:        Not affected

Versions of relevant libraries:
[pip3] dpcpp-cpp-rt==2025.3.2
[pip3] impi-rt==2021.17.2
[pip3] intel-cmplr-lib-rt==2025.3.2
[pip3] intel-cmplr-lib-ur==2025.3.2
[pip3] intel-cmplr-lic-rt==2025.3.2
[pip3] intel-opencl-rt==2025.3.2
[pip3] intel-openmp==2025.3.2
[pip3] intel-pti==0.16.0
[pip3] intel-sycl-rt==2025.3.2
[pip3] mkl==2025.3.1
[pip3] numpy==2.4.2
[pip3] oneccl==2021.17.2
[pip3] oneccl-devel==2021.17.2
[pip3] onemkl-license==2025.3.1
[pip3] onemkl-sycl-blas==2025.3.1
[pip3] onemkl-sycl-dft==2025.3.1
[pip3] onemkl-sycl-lapack==2025.3.1
[pip3] onemkl-sycl-rng==2025.3.1
[pip3] onemkl-sycl-sparse==2025.3.1
[pip3] optree==0.18.0
[pip3] tbb==2022.3.1
[pip3] tcmlib==1.4.1
[pip3] torch==2.11.0+xpu
[pip3] torchaudio==2.11.0+xpu
[pip3] torchvision==0.26.0+xpu
[pip3] triton-xpu==3.7.0
[pip3] umf==1.0.3
[conda] dpcpp-cpp-rt              2025.3.2                 pypi_0    pypi
[conda] impi-rt                   2021.17.2                pypi_0    pypi
[conda] intel-cmplr-lib-rt        2025.3.2                 pypi_0    pypi
[conda] intel-cmplr-lib-ur        2025.3.2                 pypi_0    pypi
[conda] intel-cmplr-lic-rt        2025.3.2                 pypi_0    pypi
[conda] intel-opencl-rt           2025.3.2                 pypi_0    pypi
[conda] intel-openmp              2025.3.2                 pypi_0    pypi
[conda] intel-pti                 0.16.0                   pypi_0    pypi
[conda] intel-sycl-rt             2025.3.2                 pypi_0    pypi
[conda] mkl                       2025.3.1                 pypi_0    pypi
[conda] numpy                     2.4.2                    pypi_0    pypi
[conda] oneccl                    2021.17.2                pypi_0    pypi
[conda] oneccl-devel              2021.17.2                pypi_0    pypi
[conda] onemkl-license            2025.3.1                 pypi_0    pypi
[conda] onemkl-sycl-blas          2025.3.1                 pypi_0    pypi
[conda] onemkl-sycl-dft           2025.3.1                 pypi_0    pypi
[conda] onemkl-sycl-lapack        2025.3.1                 pypi_0    pypi
[conda] onemkl-sycl-rng           2025.3.1                 pypi_0    pypi
[conda] onemkl-sycl-sparse        2025.3.1                 pypi_0    pypi
[conda] optree                    0.18.0                   pypi_0    pypi
[conda] tbb                       2022.3.1                 pypi_0    pypi
[conda] tcmlib                    1.4.1                    pypi_0    pypi
[conda] torch                     2.11.0+xpu               pypi_0    pypi
[conda] torchaudio                2.11.0+xpu               pypi_0    pypi
[conda] torchvision               0.26.0+xpu               pypi_0    pypi
[conda] triton-xpu                3.7.0                    pypi_0    pypi
[conda] umf                       1.0.3                    pypi_0    pypi

**Comment by weishi-deng** [comment]
Status: pass on the latest release whl
cmd: python benchmarks/dynamo/timm_models.py --accuracy -d xpu -n10 --backend=inductor --cold-start-latency --training  --bfloat16 --only convnextv2_nano.fcmae_ft_in22k_in1k
version:
torch                       2.11.0+xpu
torchaudio                  2.11.0+xpu
torchvision                 0.26.0+xpu
triton-xpu                  3.7.0

@mengfei25 pls help check the latest status

**Comment by daisyden** [comment]
@mengfei25 check tolerance configuration.

**Comment by jianyizh** [comment]
@mengfei25 do not change tolerance for now

**Comment by weishi-deng** [comment]
@orrangetabby17 could you help check if jianyi's PR https://github.com/pytorch/pytorch/pull/176049 can fix this issue?

**Comment by orrangetabby17** [comment]
> [@orrangetabby17](https://github.com/orrangetabby17) could you help check if jianyi's PR [pytorch/pytorch#176049](https://github.com/pytorch/pytorch/pull/176049) can fix this issue?

Sure, I can take a look.

**Comment by orrangetabby17** [comment]
Verified on B60 that [pytorch/pytorch#176049](https://github.com/pytorch/pytorch/pull/176049) can fix this issue.

**Comment by bjarzemb** [comment]
Issue is fixed with PyTorch 2.13.
